Index: python3-eventlet/tests/zmq_test.py
===================================================================
--- python3-eventlet.orig/tests/zmq_test.py
+++ python3-eventlet/tests/zmq_test.py
@@ -449,7 +449,7 @@ class TestUpstreamDownStream(tests.Limit
         self.sockets.append(sock)
         sock.bind_to_random_port("tcp://127.0.0.1")
         eventlet.sleep()
-        tests.check_idle_cpu_usage(0.2, 0.1)
+        tests.check_idle_cpu_usage(0.2, 0.2)
 
     @tests.skip_unless(zmq_supported)
     def test_cpu_usage_after_pub_send_or_dealer_recv(self):
@@ -461,14 +461,14 @@ class TestUpstreamDownStream(tests.Limit
         sub.setsockopt(zmq.SUBSCRIBE, b"")
         eventlet.sleep()
         pub.send(b'test_send')
-        tests.check_idle_cpu_usage(0.2, 0.1)
+        tests.check_idle_cpu_usage(0.2, 0.2)
 
         sender, receiver, _port = self.create_bound_pair(zmq.DEALER, zmq.DEALER)
         eventlet.sleep()
         sender.send(b'test_recv')
         msg = receiver.recv()
         self.assertEqual(msg, b'test_recv')
-        tests.check_idle_cpu_usage(0.2, 0.1)
+        tests.check_idle_cpu_usage(0.2, 0.2)
 
 
 class TestQueueLock(tests.LimitedTestCase):
